### PR TITLE
Readd missing TrajectoryConstraints functions for MoveitCommander

### LINKED ISSUE #2373: MoveGroupCommander TrajectoryConstraints uses undeclared functions from C++ wrapper
### Description

The pipeline to potentially support TrajectoryConstraints was added in 
https://github.com/ros-planning/moveit/pull/793

However, it seems like some of these functions were removed in
https://github.com/ros-planning/moveit/pull/835/files
(ex: set_trajectory_constraints_from_msg, clear_trajectory_constraints)

When currently attempting to use any of the python functions that use the removed functions, an error occurs because they no longer exist.

### PR DESCRIPTION
**Pull request by kevinshchang**
### Description

fix #2373 

Does not look like this message is used other than in very specific applications, but currently the python wrapper uses functions that do not exist, which seems like it is still worth fixing.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/ros-planning/moveit/blob/master/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://ros-planning.github.io/moveit_tutorials/doc/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp [hunk 1]
```diff
@@ -745,6 +758,11 @@ static void wrap_move_group_interface()
   move_group_interface_class.def("set_path_constraints_from_msg", &MoveGroupInterfaceWrapper::setPathConstraintsFromMsg);
   move_group_interface_class.def("get_path_constraints", &MoveGroupInterfaceWrapper::getPathConstraintsPython);
   move_group_interface_class.def("clear_path_constraints", &MoveGroupInterfaceWrapper::clearPathConstraints);
+
+  move_group_interface_class.def("set_trajectory_constraints_from_msg",
+                                 &MoveGroupInterfaceWrapper::setTrajectoryConstraintsFromMsg);
```
Comment (felixvd) [inline moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp:763]: Is there a reason this line is split in two?

File: moveit_commander/src/moveit_commander/move_group.py [hunk 2]
```diff
@@ -423,20 +423,20 @@ def clear_path_constraints(self):
...
-        c = Constraints()
+        """ Get the actual trajectory constraints in form of a moveit_msgs.msgs.TrajectoryConstraints """
+        c = TrajectoryConstraints()
         c_str = self._g.get_trajectory_constraints()
         conversions.msg_from_string(c, c_str)
         return c
 
     def set_trajectory_constraints(self, value):
-        """ Specify the trajectory constraints to be used """
+        """ Specify the trajectory constraints to be used (database not implemented)"""
```
Comment (felixvd) [inline moveit_commander/src/moveit_commander/move_group.py:433]:
```suggestion
        """ Specify the trajectory constraints to be used (setting from database is not implemented yet)"""
```
I'm not sure the docstring is the best place to keep track of this, but this should be clearer at least

**Review by felixvd — COMMENTED** [review]
Thanks for fixing this! Are you using the constraint database yourself?

**Comment by kevinshchang** [comment]
> Thanks for fixing this! Are you using the constraint database yourself?

I am not using the database myself. I actually don't use this message either, but it was just something I noticed when playing with the python interface lol.

**Review by davetcoleman — APPROVED** [review]
LGTM, would appreciate another +1 from someone else as I'm not super at Python @felixvd

**Comment by davetcoleman** [comment]
@kevinshchang could you resolve the conflicting file? sorry for the delay!

**Comment by v4hn** [comment]
I rebased and cleaned up the conflicts. Looks good (though not very useful) to me...

**Pull request merged**